src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -12,12 +12,46 @@
     GPT4_CONTEXT_WINDOW_TAG,
     GPT4_32K_CONTEXT_WINDOW_TAG,
 )
-
-from helm.benchmark.window_services.huggingface_window_service import HuggingFaceWindowService
-from helm.benchmark.window_services.gpt2_window_service import GPT2WindowService
-from helm.benchmark.window_services.remote_window_service import get_remote_window_service
-from helm.benchmark.window_services.window_service import WindowService
-from helm.benchmark.window_services.tokenizer_service import TokenizerService
+from .ai21_window_service import AI21WindowService
```

Comment:
This looks like a bad merge; I'd suggest starting with a clean branch and just adding `sagemaker_client.py`.